# Add a motion

play_motion2_msgs/Motion motion
bool overwrite
---
bool success